src/components/calculator.js

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable react/destructuring-assignment */
+import React, { useState } from 'react';
+import calculate from '../logic/calculate';
+
+function Calculator(props) {
```

Comment:
- [ ] Kindly use ES6 syntax to follow JavaScript best practices, ie. Arrow functions 👍